README.md

```diff
@@ -8,26 +8,96 @@ The goal of this project is to make a feature complete Gothic game client compat
 
-Common Gothic installation paths to be provided via `-g`:
-- "C:\Program Files (x86)\JoWooD\Gothic II"
-- "C:\Gothic II"
-- "C:\Program Files (x86)\Steam\steamapps\common\Gothic II"
-- "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"
+#### Prerequisites
 
-### Build on Linux
-#### Dependencies
+Gothic 2: Night of the Raven is required as OpenGothic does not provide any built-in game assets or scripts.
+
+Supported systems are:
+* Windows (DX12, Vulkan)
+* Linux (Vulkan)
+* MacOS (Metal)
+
+## How to play
+### Windows
+1. If not already done install Gothic. OpenGothic comes with auto-path detection if your Gothic files are in a common path.
+    * "C:\Program Files (x86)\JoWooD\Gothic II"
+    * "C:\Gothic II"
+    * "C:\Program Files (x86)\Steam\steamapps\common\Gothic II"
+2. Download OpenGothic and extract into a folder of your choice. Available options are:
+    * A [Pre-Release](https://github.com/Try/opengothic/releases/latest) (recommended)
+    * Alternatively a recent test build from [CI](https://ci.appveyor.com/project/Try/opengothic/history)
+3. Run `Gothic2Notr.exe`.
+
+   If nothing happens check if a crash file (`crash.log`) has been created and look for the line `--crashlog(std::logic_error(gothic not found!))---`. In this case OpenGothic fails to find your Gothic installation and you have to explicitly specify its location via `-g` paramter. Either you create a shortcut to `Gothic2Notr.exe` and change the target line in Properties to e.g.
+
+   `Gothic2Notr.exe -g "C:\Program Files (x86)\Steam\steamapps\common\Gothic II"`
+
+   or you can edit `Gothic2Notr.bat` and run this file instead.
+
+### Linux
+1. If not already done install Gothic via Wine/Proton or copy the game files from a Windows installation.
+2. You can download a build from [CI](https://ci.appveyor.com/project/Try/opengothic/history) and extract into a folder of your choice. Alternatively OpenGothic can be built manually. For Arch the [AUR](https://aur.archlinux.org/packages/opengothic) provides a 3rd party package.
+
+   It's also possible to run the [Pre-Release](https://github.com/Try/opengothic/releases/latest) for Windows via Wine/Proton with little performance  impact in case the other options don't suit you.
+
+3. Open `Gothic2Notr.sh`:
+
+   ```bash
+   #!/usr/bin/env bash
+   DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+   export LD_LIBRARY_PATH="$DIR:$LD_LIBRARY_PATH"
+   export DYLD_LIBRARY_PATH="$DIR:$DYLD_LIBRARY_PATH"
+   if [[ $DEBUGGER != "" ]]; then
+     exec $DEBUGGER --args "$DIR/Gothic2Notr" "$@"
+   else
+     exec "$DIR/Gothic2Notr" "$@"
+   fi
+   ```
+   Use `-g` parameter and change the line `exec "$DIR/Gothic2Notr" "$@"` to reflect your Gothic path e.g.
+
+   `exec "$DIR/Gothic2Notr" "$@" -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"`
+
+4. Run `Gothic2Notr.sh`
```

Comment:
Probably it's easier to simply 'Run `Gothic2Notr.sh` -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II" '

So step 3 can be about running with `-g` and step 4 can mention that it's possible to edit script

Reply:
Exe can't find Tempest library because export is missing. I guess then running script is better than export+command?

Reply:
no sure what do you mean by Exe. script should take care of LD_LIBRARY_PATH and forward application arguments to opengothic

Reply:
Damn it sorry was overlooking the .sh .

Reply:
Ok changed it :)